bugbug/nlp.py

```diff
@@ -37,6 +65,10 @@ def get_word_embeddings():
 
 class MeanEmbeddingTransformer(BaseEstimator, TransformerMixin):
     def __init__(self):
+        # Detect when the optional dependency is missing
```

Comment:
Nit: `# Detect when the gensim optional dependency is missing` (at some point we might split Spacy and Gensim into separate optional deps)